openshift-tools/pkg/discover-etcd-initial-cluster/initial-cluster.go

```diff
@@ -110,7 +111,9 @@ func (o *DiscoverEtcdInitialClusterOptions) Validate() error {
 }
 
 func (o *DiscoverEtcdInitialClusterOptions) Run() error {
-	_, err := os.Stat(o.MemberDir)
+
+	memberDir := filepath.Join(o.DataDir, "member")
```

Comment:
To make this transition, you'll need to make this join conditional won't you?  Otherwise you'll be appending /etcd/member/member and break the off and back on again.  Unless I messed up the original logic.